### DISCUSSION TITLE
cannot find module webpack/lib/RuleSet

### DISCUSSION THREAD

**Discussion by mukund4u** [webpack/webpack#17209] [Q&A]
# Bug report

**What is the current behavior?**
throws an error: Cannot find module 'webpack/lib/RuleSet'

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**
Webpack should build successfully

**Other relevant information:**
webpack version:  5.82.1
Node.js version:14.17.6
Operating System:Windows
Additional tools:

**Comment by alexander-akait** [comment]
There is no `webpack/lib/RuleSet` file in webpack, so you can't import it, check your loaders/plugins supports webpack v5, you can run `npm ls webpack` and look which packages use the old version of webpack
